src/MOI_wrapper.jl

```diff
@@ -89,6 +89,20 @@ function MOI.supports_constraint(
     return MOI.supports_add_constrained_variables(optimizer.dual_problem.dual_model, D)
 end
 
+function MOI.modify(optimizer::DualOptimizer{T},
+                    ::MOI.ObjectiveFunction{MOI.ScalarAffineFunction{T}},
+                    obj_change::MOI.ScalarCoefficientChange{T}) where T
+    # We must find the constraint corresponding to the variable in the objective
+    # function and change its coeeficient on the constraint.
+    ci_to_change = optimizer.dual_problem.primal_dual_map.primal_var_dual_con[obj_change.variable]
+    sense_change = MOI.get(optimizer.dual_problem.dual_model, 
+                           MOI.ObjectiveSense()) == MOI.MAX_SENSE ? one(T) : -one(T)
+    MOI.transform(optimizer.dual_problem.dual_model, 
```

Comment:
What was the type of the constraint before ? It was not `EqualTo` ?

Reply:
If it was `EqualTo` already, you can do `MOI.set(optimizer.dual_problem.dual_model, MOI.ConstraintSet(), ci_to_change, MOI.EqualTo(sense_change * obj_change.new_coefficient))`